flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/tasks/StreamTask.java

```diff
@@ -201,6 +201,8 @@
 	/** Flag to mark this task as canceled. */
```

Comment:
Commit message could also include the motivation. I'm also not sure, but I'd guess @pnowojski would declare that as a hotfix commit.

Reply:
I was more thinking to point to out the need to split it for the upcoming commits.

Reply:
@rkhachatryan ping?

Reply:
updated commit message